src/seedu/addressbook/AddressBook.java

```diff
@@ -203,13 +203,17 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        executeUserCommand();
+    }
+
+	public static void executeUserCommand() {
+		while (true) {
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically